research/src/pages/popup/popup.proposal.mdx

```diff
@@ -12,6 +12,10 @@ import '../../styles/spec.css'
 - Mason Freed (Google)
 - Melanie Richards (Microsoft)
 
+# Note: This is an outdated proposal!
```

Comment:
Can we rename this file then to `popup.proposal-v1.mdx` and your explainer to `popup.proposal.md` so that everything links up as expected? Then link to the prior proposal at the base of the doc under resources.

Reply:
Sure, I'll take care of that. What would you like to rename `popup.research.explainer.mdx`? (The naming in this folder was already pretty confusing, so I'm really happy to try to clarify it a bit.) Before this PR, here are the files and their contents:

- [`popup.research.explainer.mdx`](https://open-ui.org/components/popup.research.explainer): The original `<popup>` element **explainer**. This should be the thing that gets renamed to "v1", I think. 
- [`popup.research`](https://open-ui.org/components/popup.research): The anatomy of a popup. This should stay as-is.
- [`popup.proposal.mdx`](https://open-ui.org/components/popup): A set of draft spec text for the `<popup>` element proposal, which should likely also get renamed to something like "v1".

Now I just need to figure out how the routing on this site works, e.g. turning `popup.proposal.mdx` into just `/components/popup`, while turning `popup.research.explainer.mdx` into `/components/popup.research.explainer`.